scripts/ci/in_container/run_ci_tests.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+# Bash sanity settings (error on exit, complain for undefined vars, error when pipe fails)
+set -euo pipefail
+
+MY_DIR=$(cd "$(dirname "$0")"; pwd)
+
+if [[ ${AIRFLOW_CI_VERBOSE:="false"} == "true" ]]; then
+    set -x
+fi
+
+. ${MY_DIR}/_check_in_container.sh
+
+# any argument received is overriding the default nose execution arguments:
+NOSE_ARGS=$@
+
+echo "Initializing the DB"
+yes | airflow initdb || true
+airflow resetdb -y
+
+kinit -kt ${KRB5_KTNAME} airflow
```

Comment:
````
kinit -kt "${KRB5_KTNAME}" airflow
````
Double quote to prevent globbing and word splitting.

Reply:
Yep